From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Tue, 20 Dec 2022 01:05:38 -0500
Subject: [PATCH] filter: python: Fix polezero_plot.py compatibility with numpy
 1.24

np.float as an alias for float has been deprecated and removed, and
np.float_ or np.float64 is undoubtedly what was wanted here anyway.

Signed-off-by: Ryan Volz <ryan.volz@gmail.com>
---
 gr-filter/python/filter/gui/polezero_plot.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-filter/python/filter/gui/polezero_plot.py b/gr-filter/python/filter/gui/polezero_plot.py
index 859cd5c2d..9d6db915e 100644
--- a/gr-filter/python/filter/gui/polezero_plot.py
+++ b/gr-filter/python/filter/gui/polezero_plot.py
@@ -11,7 +11,7 @@ from PyQt5 import Qt, QtCore
 from math import sin, cos, pi
 import pyqtgraph as pg
 from numpy import zeros
-from numpy import float as Float
+from numpy import float_ as Float
 from numpy import vectorize
 from numpy import delete
 
-- 
2.38.1

